forge-gui/res/adventure/common/world/points_of_interest.json

```diff
@@ -3130,6 +3130,18 @@
 		"Planeswalker"
 	]
 },
+{
+	"name": "UnhallowedAbbey",
+	"type": "dungeon",
+	"count": 1,
+	"spriteAtlas": "maps/tileset/buildings.atlas",
+	"sprite": "MageTower",
```

Comment:
would (maybe altered ?) "monastery" be a better fit ?

Reply:
It was actually supposed to be Monastery, thanks for catching that! ^^